es-ds-docs/package.json

```diff
@@ -10,8 +10,8 @@
     "postinstall": "nuxt prepare"
   },
   "dependencies": {
-    "@energysage/es-bs-base": "^2.0.4",
-    "@energysage/es-ds-components": "^0.0.1",
+    "@energysage/es-ds-components": "^3.0.0-alpha.6",
```

Comment:
We might want to consider reverting this to `^0.0.1` since it's independent of the previous packages (same with `es-ds-styles`)

Reply:
The one catch with that is that we would then be unable to ever have a release of es-ds-components or es-ds-styles with these versions that are already published because of [a restriction of NPM](https://docs.npmjs.com/cli/v10/commands/npm-unpublish). In any case we can certainly decide to rename things or accept that limitation if we prefer. Maybe worth a followup conversation